applications/matter_weather_station/README.rst

```diff
@@ -175,6 +183,40 @@ The ``build_thingy53_nrf5340_cpuapp`` parameter specifies the output directory f
 
       File not found: ./ncs/nrf/applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/prj_shell.conf
 
+Pre-prepared factory data support
+=================================
```

Comment:
```suggestion
=================================
---------------------------------
```
I'd nest this under Selecting a build type, because it is required to select the prj_factory_data.conf here.